khal/khalendar/event.py

```diff
@@ -638,15 +638,13 @@ def __init__(self, *args, **kwargs):
         try:
             starttz = getattr(self._vevents[self.ref]['DTSTART'].dt, 'tzinfo', None)
         except KeyError:
-            logger.fatal(
-                "Cannot understand event {} from calendar {},\n you might want to"
-                "file an issue at https://github.com/pimutils/khal/issues"
-                "".format(kwargs.get('href'), kwargs.get('calendar'))
-            )
+            msg = "Cannot understand event {} from " \
+                  "calendar {}, you might want to file an issue at " \
```

Comment:
Intentation should remain the same (also, as a convention, indentation should always be four spaces).